app/controllers/admin/visit_requests_controller.rb

```diff
@@ -1,18 +1,37 @@
 module Admin
   class VisitRequestsController < ::Admin::BaseController
-    helper_method :visit_requests, :event, :visitors_ids, :confirmed_ids, :applied_ids,
+    helper_method :form, :visit_requests, :event, :visitors_ids, :confirmed_ids, :applied_ids,
                   :pending_ids, :approved_ids
 
-    before_action :only_supervisor!
+    before_action :only_supervisor!, :set_breadcrumbs
 
-    def index
+    def create
+      @form = InviteByEmailForm.new(visit_request_params)
```

Comment:
You already have `form` method. Just pass `visit_request_params` to the constructor.